plugins/modules/postgresql_lang.py

```diff
@@ -12,9 +12,11 @@
 ---
 module: postgresql_lang
 short_description: Adds, removes or changes procedural languages with a PostgreSQL database
+deprecated:
+- removed_in: "4.0.0"
+- why: As of PostgreSQL 9.1, most procedural languages have been made into extensions.
+- alternative: Use M(postgresql_ext) instead.
```

Comment:
```suggestion
- alternative: Use M(community.postgresql.postgresql_ext) instead.
```
Using a FQCN is a [requirement](https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-and-other-format-macros-within-module-documentation) with this type of macros